### DISCUSSION TITLE
API -> save_user -> send_mail problem

### DISCUSSION THREAD

**Discussion by MeInTheGalaxie** [chamilo/chamilo-lms#5850] [General]
Good morning,

I am testing the API and it works well for creating a user, but I am encountering a problem sending an email when creating the user. It is indicated available on camilo 1.11.28, I am in 1.11.26, probably for that. I was able to test the mail() function and it works perfectly. Where can I find the 1.11.28 update to be able to use the send_mail = 1 parameter ?

**Comment by Kaneda-1** [comment]
Hi @SSIReseauCS 

https://github.com/chamilo/chamilo-lms/tree/1.11.x

**For developers and testers only**
This section is for developers only (or for people who have a good reason to use a development version of Chamilo), in the sense that other people will not need to update their Chamilo portal as described here.

Updating code
To update your code with the latest developments in the 1.11.x branch, go to your Chamilo folder and type:

git pull origin 1.11.x

......
